saleor/plugins/admin_email/tests/test_csv_emails.py

```diff
@@ -0,0 +1,87 @@
+#
+# @mock.patch("saleor.csv.emails.send_templated_mail")
```

Comment:
Maybe instead of comment-out this test we can use `@pytest.mark.skip`?

Reply:
Not sure if this will solve the problem, the mock paths have been changed.

Reply:
If you add `skip` on those tests, Pytest didn't validate mocked paths.